bin/hammer

```diff
@@ -92,6 +94,8 @@ HammerCLI::Settings.load({
     :ssl_with_basic_auth => preparser.ssl_with_basic_auth?
   }})
 
+HammerCLI::Settings.load({:use_defaults => preparser.use_defaults?}) unless preparser.use_defaults?.nil?
```

Comment:
Isn't it a duplication of https://github.com/theforeman/hammer-cli/pull/293/files#diff-aca6a835822ab9b99782be0a5f01f8ddR87 ? If it's a shorcut for `Settings.get(:_params, :use_defaults)`, why keep it under `_params` then?

Reply:
@ofedoren Exactly. I had a reason to have it there but it was bad idea indeed. Removed.